examples/guides/messaging/message-broadcasting/guide/database_writer.bal

```diff
@@ -0,0 +1,71 @@
+import ballerina/jms;
+import ballerina/log;
+import ballerina/http;
+import ballerina/io;
+import ballerina/mysql;
+import ballerina/sql;
+
+//JMS listener listening on topic
+listener jms:TopicSubscriber subscriberEndpoint = new({
+        initialContextFactory: "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+        providerUrl: "tcp://localhost:61616",
+        acknowledgementMode: "AUTO_ACKNOWLEDGE"   //remove message from broker as soon as it is received
+    }, topicPattern = "appointmentTopic");
+
+//MySQL DB client configuration
+mysql:Client healthcareDB = new({
+        host: "localhost",
+        port: 3306,
+        name: "healthcare",
+        username: "wso2",
+        password: "wso2",
+        dbOptions: { useSSL: false }
+    });
+
+service jmsListener on subscriberEndpoint {
+    //invoked upon JMS message receive
+    resource function onMessage(jms:TopicSubscriberCaller consumer,
+    jms:Message message) {
+        //receive message as a text
+        var messageText = message.getTextMessageContent();
+        if (messageText is string) {
+            io:StringReader sr = new(messageText, encoding = "UTF-8");
+            json jasonMessage = checkpanic sr.readJson();
+            //write to database
+            writeToDB(jasonMessage);
+
+        } else {
+            log:printInfo("Error occurred while reading message " + messageText.reason());
```

Comment:
Isn't it more appropriate to have log:printError ?

Reply:
Corrected the statement.